extension/manifest.json

```diff
@@ -8,11 +8,20 @@
     "<all_urls>"
   ],
   "background": {
-    "scripts": ["event-page.js"],
-    "persistent": false
+    "page": "event-page.html",
+    "persistent": true
```

Comment:
Why does this need to be persistent?

Reply:
That's a good question. There was a bug that caused it to be persistent, and then I hung on to it. I've filed #443.